content/en/docs/concepts/configuration/secret.md

```diff
@@ -31,7 +35,10 @@ access, or anyone with access to Kubernetes' underlying data store, etcd. In
 order to safely use Secrets, it is recommended you (at a minimum):
 
 1. [Enable Encryption at Rest](/docs/tasks/administer-cluster/encrypt-data/) for Secrets.
-2. [Enable or configure RBAC rules](/docs/reference/access-authn-authz/authorization/) that restrict reading and writing the Secret. Be aware that secrets can be obtained implicitly by anyone with the permission to create a Pod.
+2. [Enable or configure RBAC rules](/docs/reference/access-authn-authz/authorization/) that
+   restrict reading and writing the Secret. Be aware that secrets can be obtained
```

Comment:
restrict reading from and writing to the Secret.

Reply:
This change is a subset of the changes from PR #24169 (a PR that has proved too big to review).

I will follow this up with further PRs to incorporate the other changes that #24169 suggests. Anyone else can make those follow-ups; it doesn't have to be me.